web-client/src/environments/environment.bhutan-prod.ts

```diff
@@ -25,6 +25,10 @@ export const environment: Environment = {
   commissionPercentage: 0.0,
   tokenIssuer: 'rKitZq2qY8REiq7xvY3MfaqMhQkMekJtK',
   tokenSymbol: 'BTN',
-  xrpIssuer: 'rpJv16Qmn2rQP6UC6UFsNRnVy5arkQihPP',
-  hideXrpBalance: true,
+  xrpIssuer: 'rpd17stoaELtzuhxEn4eAQcipXy2w8DEXQ',
+  hideXrpBalance: false,
```

Comment:
The `hideXrpBalance` variable should be set to true for the Bhutan & Palau environments.